test/zzz_od/operation/test_choose_predefined_team.py

```
@@ -0,0 +1,101 @@
+        [
+            {
+                'name': '新队伍三',
+                'auto_battle': '全配队通用',
+                'agent_id_list': ['unknown', 'unknown', 'unknown'],
+            }
+        ],
+    )
+
+
+def test_update_team_name_by_idx_removes_whitespace() -> None:
+    """按序号同步名称时，保存前移除 OCR 结果中的空白。"""
```

Ruff (0.16.0)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)